### PR TITLE
VMO-4367 + VMO-4362 Fix when duplicating multiple blocks doesn't merge metadata

### PR DESCRIPTION
**Pull request by bzabos**
This fixes Default showing up on multi-duplicate.

---

Screen Shot 2021-07-27 at 10 09 19 PM

### CODE REVIEW

File: src/store/flow/flow.ts [hunk 1]
```diff
@@ -361,14 +361,15 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
 
     // Set UI positions
     // TODO: remove this once IBlock has vendor_metadata key
```
Comment (safydy-r) [inline src/store/flow/flow.ts:363]: @bzabos , I think we missed to remove this TODO, can we take the opportunity to remove this ?

File: src/store/flow/flow.ts [hunk 2]
```diff
@@ -361,14 +361,15 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
...
-    duplicatedBlock.vendor_metadata = {
+    merge(duplicatedBlock.vendor_metadata, {
```
Comment (safydy-r) [inline src/store/flow/flow.ts:364]: Neat solution, thanks 💯

**Review by safydy-r — APPROVED** [review]
Thanks @bzabos , very clear solution.
I left one minor request

File: src/store/flow/flow.ts (hunk 2 above)
Comment (bzabos) [reply src/store/flow/flow.ts:364]: 😻 I liked how clean this changeset was!

**Pull request merged**